integration-tests/src/test/java/uk/gov/di/authentication/api/StartIntegrationTest.java

```diff
@@ -383,6 +320,34 @@ private SignedJWT createSignedJWT(KeyPair keyPair, State state) throws JOSEExcep
         return signedJWT;
     }
 
+    private void verifyStandardClientInformationSetOnResponse(
+            StartResponse startResponse, Scope scope, State state) {
+        assertThat(startResponse.getClient().getClientName(), equalTo(TEST_CLIENT_NAME));
```

Comment:
We're calling `.getClient()` in every line for this set of verifications, so a refactoring might be to make it take a `Client` rather than a `StartResponse`, and use `startResponse.getClient()` at the method's callsite instead.

Same applies for `verifyStandardUserInformationSetOnResponse` below.